src/transactions/TransactionFrame.cpp

```diff
@@ -296,8 +296,13 @@ TransactionFrame::commonValid(SignatureChecker& signatureChecker,
         return false;
     }
 
+    auto balanceAfter =
```

Comment:
add a comment to explain that fees are captured in a different step when applying